packages/orbit-design-tokens/.npmrc

```diff
@@ -0,0 +1,2 @@
+scope=@kiwicom
+access=public
```

Comment:
Deleting this file should be fine, but just in case add `"private": false` to `package.json`.